src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementServiceTest.kt

```diff
@@ -49,26 +70,93 @@ class MovementServiceTest {
     assertThat(moves).isEqualTo(
       listOf(
         Movement(
+          id = "1",
           firstName = "JIM",
           lastName = "SMITH",
           dateOfBirth = LocalDate.of(1991, 7, 31),
           prisonNumber = "A1234AA",
           pncNumber = "99/123456J",
           date = LocalDate.of(2021, 1, 2),
           fromLocation = "MDI",
-          moveType = MoveType.PRISON_TRANSFER
+          moveType = PRISON_TRANSFER
         ),
         Movement(
+          id = "1",
           firstName = "First",
           lastName = "Last",
           dateOfBirth = LocalDate.of(1980, 2, 23),
           prisonNumber = "A1278AA",
           pncNumber = "1234/1234589A",
           date = LocalDate.of(2021, 1, 2),
-          moveType = MoveType.PRISON_TRANSFER,
+          moveType = PRISON_TRANSFER,
           fromLocation = "MDI"
         )
       )
     )
```

Comment:
This test appears to create two Movement objects, `m1` and `m2` say, then assert that the return value from the getMovements function returns `listOf(m1, m2)`.  
The test would be less verbose and easier to understand if you created `m1` and `m2`...